package.json

```diff
@@ -54,12 +54,12 @@
       "fs": "github:jspm/nodelibs-fs@^0.2.0-alpha",
       "path": "github:jspm/nodelibs-path@^0.2.0-alpha",
       "process": "github:jspm/nodelibs-process@^0.2.0-alpha",
-      "rxjs": "npm:rxjs@^5.0.0-beta.6",
+      "rxjs": "npm:rxjs@5.0.0-beta.6",
       "stream": "github:jspm/nodelibs-stream@^0.2.0-alpha",
       "string_decoder": "github:jspm/nodelibs-string_decoder@^0.2.0-alpha",
       "util": "github:jspm/nodelibs-util@^0.2.0-alpha",
       "vm": "github:jspm/nodelibs-vm@^0.2.0-alpha",
-      "zone.js": "npm:zone.js@0.6.12"
+      "zone.js": "npm:zone.js@^0.6.6"
```

Comment:
Ehm... I don't think this is intended (it seems to happens on `jspm install`, for no serious reason...). We should stick to fixed versions here for now, especially since issues like https://github.com/angular/zone.js/issues/407 are raising far too often even for minor versions.